content/general/concepts/vim/vim.md

```diff
@@ -0,0 +1,35 @@
+---
+Title: 'Vim'
+Description: 'Vim is a free, open source, and customizable text editor for Linux.'
+Subjects:
+  - 'Bash/Shell'
+  - 'Developer Tools'
+Tags:
+  - 'Vim'
+  - 'Linux'
+  - 'Developer Tools'
+CatalogContent:
+  - 'paths/computer-science'
+---
+
+[Vim](https://www.vim.org/) is a free, open source, and customizable text editor that can be run on Windows, Mac, Unix, Linux, or Android. It can be run directly from the command line interface or from it's GUI interface. It is based on 'vi', another text editor that was created for the unix operating system. Many plugins are available to add to the many built in features. Developers will find it useful for working on source code or scripting files. Vim is also very convenient for editing configuration files when working from the terminal.
+
+## Features
+
+- Text completion
+- File comparison and merging
+- Search for regular expressions
+- Can perform complicated tasks using only commands
+- Open multiple tabs
+- Create macros
+- Save sessions
+
+## Launching Vim
+
+```bash
+vim filename.txt
+```
+
```

Comment:
Maybe a short sentence or two below this snippet explaining what the command is doing? Is it opening a new Vim window with the contents of `filename.txt`?